scripts/test-server.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
If I remove the xvfb it all crashes in an lxc--not sure why. And yes, that's what it's for.